root/etc/s6-overlay/s6-rc.d/init-permissions-config/run

```diff
@@ -4,8 +4,10 @@
 # permissions
 lsiown -R abc:abc \
     /config
-chmod -R 0644 /etc/logrotate.d
 chmod -R +r /config/log
+if [[ -z ${LSIO_READ_ONLY_FS} ]]; then
+    chmod -R 0644 /etc/logrotate.d
+fi
```

Comment:
Question: does logrotate work in non-root/RO?

Reply:
Seems to be (my swag instance gets very little traffic as it's only for testing):
```
# cat /config/log/logrotate.status
logrotate state -- version 2
"/var/log/acpid.log" 2024-2-16-2:0:0
"/config/log/nginx/access.log" 2024-12-18-2:0:0
"/var/log/php83/*.log" 2024-1-30-2:0:0
"/config/log/nginx/error.log" 2024-12-18-2:0:0
"/config/log/letsencrypt/letsencrypt.log" 2024-12-29-2:0:0
"/config/log/letsencrypt/renewal.log" 2024-12-29-2:0:0
"/config/log/fail2ban/fail2ban.log" 2024-12-29-2:0:0
"/config/log/php/error.log" 2024-12-29-2:0:0
"/config/log/nginx/unauthorized.log" 2023-2-14-2:0:0
```

Reply:
I went digging and found this section seems to duplicate behavior that's in the nginx base image
https://github.com/linuxserver/docker-baseimage-alpine-nginx/blob/89f7226f986fd142b9b650f8bdf0c691fd88c02c/root/etc/s6-overlay/s6-rc.d/init-permissions/run#L9 (and the surrounding lines)

Should we just remove it from SWAG? I don't see a similar line in other downstream images using the nginx base.

Reply:
It's probably something we moved to the base and forgot to remove from swag. I'll drop it.

Reply:
Done in [`29bd5fe` (#526)](https://github.com/linuxserver/docker-swag/pull/526/commits/29bd5fe1b72346e4c92089b31ec3123c95b9ec88)